MOV-AI/containers-haproxy
# DP-1380 : align dev branch with latest addition for certificates,  ros-tools and influxdb

### Context

In EE 2.4.1, haproxy configuration is overwritten dynamically by mounting a templated `haproxy.cfg` file inside the container. Still this PR tries and align the `main` branch to be iso functional as the `releases/1.1.x`

## Content

- port of #50 
- port of #47 

### Test
- build and tag the docker images
- use this image in a EE 2.4.1 robot
- check all old and new endpoints: `/`, `/monitoring`, `/influxdb`, ...
- 
---

- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
